src/app/components/movies/movies.component.sass

```diff
@@ -0,0 +1,52 @@
+$movie-card--height: 450px
+$movie-card--width: 300px
+$movie-card--border-radius: 0.05 * $movie-card--height
```

Comment:
what's the meaning of this line?

Reply:
It is 5% border radius wich is relative to mesurement of movie card. It is needed for use on overview element and automatic change it in case of changing movie card's height in mobile version, for example